public/css/login.css

```diff
@@ -0,0 +1,59 @@
+* {
+  padding: 0%;
+  margin: 0;
+}
+
+body {
+  font-family: Roboto;
+  background-color: #364150;
+}
+.header {
+  display: flex;
+  justify-content: center;
+  margin-top: 3%;
```

Comment:
Do not solve your problems using margin, and definitely not using relative margin to screen with. I don't like having the login page changing position up and down depending on my screen's width.